### PR TITLE
feat: improve app startup time

### PR DESCRIPTION
**Pull request by iGroza**
Fixes #

## Proposed Changes

- improve app startup time

## Checklist

- [x] I have read the rules of [**contribution**](https://github.com/haqq-network/haqq-wallet/blob/main/docs/contribution.md#important-steps).

### CODE REVIEW

File: src/helpers/prefetch-wallet-card-images.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import {Image} from 'react-native';
+
+import {Wallet} from '@app/models/wallet';
+import {getPatternName} from '@app/utils';
+
+export async function prefetchWalletCardImages() {
+  try {
+    await Promise.all(
```
Comment (ragozin-nikita) [inline src/helpers/prefetch-wallet-card-images.ts]: Do we have `Promise.allSettled` support?

Comment (iGroza) [reply src/helpers/prefetch-wallet-card-images.ts]: yes, it's better. Already replace to allSettled

File: src/event-actions/on-app-initialized.ts [hunk 2]
```diff
@@ -1,12 +1,12 @@
-import {onCheckAvailability} from '@app/event-actions/on-check-availability';
 import {onRemoteConfigSync} from '@app/event-actions/on-remote-config-sync';
+import {EthRpcEndpointAvailability} from '@app/helpers/eth-rpc-endpoint-availability';
 import {RemoteConfig} from '@app/services/remote-config';
 
 /**
  * @description Called first when the app is initialized. Load remote config and check availability endpoints.
  */
 export async function onAppInitialized() {
+  EthRpcEndpointAvailability.checkEthRpcEndpointAvailability();
```
Comment (vivalaakam) [inline src/event-actions/on-app-initialized.ts:9]: In config can come new endpoints, for this reason check should be after fetching the config

File: src/app.tsx [hunk 3]
```diff
@@ -66,8 +66,8 @@ export const App = () => {
         if (app.onboarded) {
           await app.init();
           await migrationWallets();
-          // MobX stores migration
-          await Promise.all([
+          // MobX stores migratio
```
Comment (ragozin-nikita) [inline src/app.tsx:69]:
```suggestion
          // MobX stores migration
```

File: src/debug-vars.ts [hunk 4]
```diff
@@ -38,6 +42,11 @@ const debug: DebugVars = {
   enableHttpErrorDetails: true,
   enableAsyncEventEmitterLogs: false,
   enableAwaitJsonRpcSignLogger: true,
+  enableMockWallets: true,
+  mockWalletsAddresses: [
+    '0x65221cede3abdd03e377e3a4ce84e14fcd497919',
```
Comment (ragozin-nikita) [inline src/debug-vars.ts:47]: We have hardcoded wallets?

File: src/services/remote-config/remote-config-service.ts [hunk 5]
```diff
@@ -22,61 +24,67 @@ function getCachedConfig() {
...
+      if (RemoteConfigService.instance.isInited) {
+        return RemoteConfigService.instance.getAll();
       }
-
+      this.startInitialization();
       const appInfo = await getAppInfo();
       const config = await Backend.instance.getRemoteConfig(appInfo);
 
-      logger.log('config', config);
+      // logger.log('config', config);
```
Comment (ragozin-nikita) [inline src/services/remote-config/remote-config-service.ts:44]: Unused code

File: src/services/wallet-connect.ts [hunk 6]
```diff
@@ -74,7 +85,7 @@ export class WalletConnect extends EventEmitter {
         metadata: {
           name: 'HAQQ Wallet',
           description: 'HAQQ Wallet for WalletConnect',
-          url: 'https://walletconnect.com/',
+          url: 'https://islamiccoin.net/',
```
Comment (ragozin-nikita) [inline src/services/wallet-connect.ts:88]: Are you sure?

**Review by ragozin-nikita — CHANGES_REQUESTED** [review]

File: src/services/wallet-connect.ts (hunk 6 above)
Comment (iGroza) [reply src/services/wallet-connect.ts:88]: yes

**Review by ragozin-nikita — APPROVED** [review]

**Pull request merged**